### DISCUSSION TITLE
Tagging Träwelling users in Travel

### DISCUSSION THREAD

**Discussion by juengerjustus** [Traewelling/traewelling#2356] [Ideas / Feature Requests]
I think it would be useful to be able to tag Träwelling users in posts when you are travelling with your group, for example.
Example: "@juengerjustus"
The link then leads to the profile of the other user. The user receives a notification when they are tagged.

**Comment by HerrLevin** [comment]
Thank you for your idea! Since it's relatively knew (yesterday) it's not well-known yet but this feature already exists. 😊 https://github.com/Traewelling/traewelling/pull/2336

**Reply by juengerjustus** [reply]
Cool, thank you!

**Reply by juengerjustus** [reply]
When will the feature be available on Träwelldroid?

**Reply by MrKrisKrisu** [reply]
Träwelldroid is a third party application and not maintained by us. This would be a question to the official repository of Träwelldroid:
https://github.com/Traewelldroid/traewelldroid

**Comment by HerrLevin** [comment]
(Reopened it since the notifications aren't done yet. Thanks @jeyemwey)

**Comment by zoneh5** [comment]
Thank you for sharing after all i found one of the best choice already i explore much of blogs for [Qatar Alive](https://qataralive.com/),
